src/Sulu/Bundle/AdminBundle/Resources/js/components/Select/Select.js

```diff
@@ -145,7 +149,11 @@ class Select<T> extends React.Component<Props<T>> {
                                 menuRef={setPopoverElementRef}
                                 style={popoverStyle}
                             >
-                                {clonedChildren}
+                                {clonedChildren.length > 0 ? clonedChildren : (
+                                    <Option disabled={true}>
+                                        {translate(noOptionsText)}
```

Comment:
I was actually just suggesting this option if we want to leave the other text. I think if we do not use this option yet, I would not implement it.

And btw, in cases like this we decided to translate the text in the caller component, so that we have only to send the string in this component. Makes it more flexible when we might have some phrases that are not translatable.